Carbon.IIS/Functions/Get-CIisCollectionItem.ps1

```diff
@@ -0,0 +1,61 @@
+function Get-CIisCollectionItem
+{
+    <#
+    .SYNOPSIS
+    Gets the items from an IIS configuration collection.
+
+    .DESCRIPTION
+    The `Get-CIisCollectionItem` function gets the items from an IIS configuration element collection. Pass the
+    collection's IIS configuration section path to the `SectionPath` parameter. If the configuration section is actually
+    the parent element of the the collection, pass the name of the child element collection to the `CollectionName`
+    parameter. To get the section for a specific site, directory, application, or virtual directory, pass its location
+    path to the `LocationPath` parameter.
+
+    You can pass an instance of a `[Microsoft.Web.Administration.ConfigurationElement]` to the `ConfigurationElement`
+    parameter to return that collection element's items, or, with the `CollectionName` parameter, get a named collection
+    under that configuration element.
+
+    This function returns configuration element collection items. To get the collection object itself, use
+    `Get-CIisCollection`.
+
+
+    .EXAMPLE
+    $items = Get-CIisCollectionItem -SectionPath 'system.webServer/httpProtocol' -CollectionName 'customHeaders'
+
+    Demonstrates how to get the custom HTTP headers from the 'customHeaders' collection, which is a child of the
+    "system.webServer/httpProtocol" configuration section.
+    #>
+    [CmdletBinding()]
+    param(
+        # The `[Microsoft.Web.Administration.ConfigurationElement]` object whose collection items to get, or the parent
+        # element of the collection whose items to get. If this is the parent element, pass the name of the child
+        # element collection to the `CollectionName` parameter.
+        [Parameter(Mandatory, ParameterSetName='Direct')]

```

Comment:
Use parameter set name `ByConfigurationElement` to be consistent with other functions.